torch/csrc/jit/mobile/import.cpp

```diff
@@ -216,8 +216,14 @@ void parseMethods(
       function->append_constant(constant);
     }
 
+    static const c10::QualifiedName classPrefix = "__torch__.torch.classes";
     for (const auto& t : types_list) {
-      function->append_type(c10::parseType(t.toStringRef()));
+      c10::QualifiedName qn(t.toStringRef());
+      if (classPrefix.isPrefixOf(qn)) {
+        function->append_type(getCustomClass(qn.qualifiedName()));
+      } else {
+        function->append_type(c10::parseType(t.toStringRef()));
+      }
     }
 
     function->set_register_size(register_size);
```

Comment:
I guess, in the end, the check for "torch" in the type_resolver did not need to be added?

Reply:
That would be the target. It's split to the second PR (will upload it soon). I'm looking for a way to not adding the "torch" type check.